proxy/api/src/project.rs

```diff
@@ -113,17 +177,22 @@ pub fn list_projects(api: &coco::Api) -> Result<Vec<Project>, error::Error> {
 pub fn list_projects_for_user(
     api: &coco::Api,
     user: &coco::Urn,
-) -> Result<Vec<Project>, error::Error> {
-    let all_projects = list_projects(api)?;
+) -> Result<Vec<Tracked>, error::Error> {
```

Comment:
I wouldn't use `Tracked` here because we're not ensuring that these projects have `signed_refs`